test/data/schemas.test.ts

```diff
@@ -226,18 +225,6 @@ test('state incentives JSON files match schemas', async tap => {
   });
 });
 
-test('All items used in IRA incentives have urls available', async tap => {
```

Comment:
Is this test case no longer valid?

Reply:
This invariant started being enforced by JSON Schema validation in #456 , so the test was already redundant. Now `incentive.item` is no longer a thing, so rather than update the test I deleted it.